keyboards/alectardy98/alpinenoon/readme.md

```diff
@@ -0,0 +1,14 @@
+# Gvalchca x Alectardy98 Alpinenoon
+
+Alps and MX 65% +10 PCB with ANSI and iso Enter support. To update the firmware connect the PCB to your computer and press Reset button on the back of the pcb.
```

Comment:
Could you separate out the firmware part, so it's more clear how to update the firmware?